Explorer/Assets/DCL/PluginSystem/World/SceneUIPlugin.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Arch.SystemGroups;
+using DCL.Optimization.Pools;
+using DCL.PluginSystem.World.Dependencies;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Systems;
+using ECS.ComponentsPooling.Systems;
+using ECS.LifeCycle;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace DCL.PluginSystem.World
+{
+    public class SceneUIPlugin : IDCLWorldPluginWithoutSettings
+    {
+        private readonly UIDocument canvas;
+        private readonly IComponentPoolsRegistry componentPoolsRegistry;
+
+        public SceneUIPlugin(
+            UIDocument canvas,
+            ECSWorldSingletonSharedDependencies singletonSharedDependencies)
+        {
+            this.canvas = canvas;
+            SetupCanvas();
+
+            componentPoolsRegistry = singletonSharedDependencies.ComponentPoolsRegistry;
+            componentPoolsRegistry.AddComponentPool<VisualElement>();
```

Comment:
As I mentioned in the thread release behavior that would disable and disconnected elements from the parent is missing

Reply:
Done!